tests/Aspire.Playground.Tests/AppHostTests.cs

```diff
@@ -137,22 +131,47 @@ public async Task TestEndpointsReturnOk(TestEndpoints testEndpoints)
                 Assert.True(HttpStatusCode.OK == response.StatusCode, $"Endpoint '{client.BaseAddress}{path.TrimStart('/')}' for resource '{resource}' in app '{Path.GetFileNameWithoutExtension(appHostPath)}' returned status code {response.StatusCode}");
             }
         }
+        if (testEndpoints.WhenReady != null)
+        {
+            foreach (var resource in resourceEndpoints.Keys)
+            {
+                await testEndpoints.WhenReady(app, appHostPath, _testOutput);
+            }
```

Comment:
Why are we firing the `WhenReady` for each `resource`, but not passing the `resource` into the method?

Reply:
Actually, yeah, this shouldn't be invoke once-per-endpoint, it should be invoked just once.